server/status_test.go

```diff
@@ -428,6 +432,9 @@ func TestStatusLocalLogs(t *testing.T) {
 		if testCase.StartTimestamp > 0 {
 			fmt.Fprintf(&url, "endtime=%d&", testCase.EndTimestamp)
 		}
+		if len(testCase.Pattern) > 0 {
```

Comment:
right, but where is the assertion that the returned logs actually match that pattern?